src/WebJobs.Script/Rpc/Configuration/GenericWorkerProvider.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.Abstractions;
+using Microsoft.Azure.WebJobs.Script.Config;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    internal class GenericWorkerProvider : IWorkerProvider
```

Comment:
Make this abstract base class with default implementations for the IWorkerProvider.  Node, Python and other languages could just the defaults, Java can override TryConfigureArguments.

Reply:
I'm not sure I like this approach. I don't want "Java" or any language specific behavior in the host. My goal is to extend the `worker.config.json` behavior to allow for the same checking conditions. This PR's scope is to just enable the Python work, but I'll send another PR which will have updates to remove the Node and Java providers. We need to discuss how best to define some of the Java specific behavior today, so I'll tackle that in a separate PR to keep the scope small.

Reply:
Never mind then. I thought we were going to keep Node and Java in the host